docs/07-developers.md

```diff
@@ -2,57 +2,75 @@
 
 Requirements:
 
-* Node.js current (8.x)
-* yarn >= 1.0.2
+- Node.js current (8.x)
+- yarn >= 1.0.2 || current stable (1.10.1)
```

Comment:
Actually, it's now better to use yarn >= 1.10.x since it adds the "integrity" hashes to the lockfile